tests/test_taskConfigs.py

```diff
@@ -54,7 +54,7 @@ def test_amx_config(self):
         expected.faint_mag_cut = 23.8
         expected.threshAD = 19.63
         expected.threshAF = 11.55
-        expected.bins = [0.5, 1.6, 3.1, 2.8]
+        expected.bins = [0.5, 1.6, 2.8, 3.1]
```

Comment:
How did this even work before?  Or is it just checking unvalidated configs and not actually running anything....

Reply:
All the `check_config` call was doing was comparing the values in the `config` with the values in `expected`; `listCheck=isSorted` in the `bins` field only gets called when `validate` gets called, so the `bins` were never being sorted. I don't know what the intent of these tests was, but they don't seem to be testing much more than "can you copy one config to another", and this isn't the right place for that.